### ISSUE TITLE
Thoughts

### ISSUE THREAD

**Issue by CHUNYUWANG** [CHUNYUWANG/attackNetwork#3]
Neuron coverage: transform current input; can we calculate the input by gradient descent?

**Comment by CHUNYUWANG** [comment]
DeepXplore

**Comment by CHUNYUWANG** [comment]
Errors transfer across models; models trained across epochs
